wire-ios-transport/Source/TransportSession/ZMTransportSession.m

```diff
@@ -459,6 +459,10 @@ - (void)didCompleteRequest:(ZMTransportRequest *)request data:(NSData *)data tas
     ZMTransportResponse *response = [self transportResponseFromURLResponse:httpResponse data:data error:transportError apiVersion:request.apiVersion];
     [self.remoteMonitoring logWithResponse:httpResponse];
 
+    /// If the label is "federation-not-available" we should not try again this request
+    if ([[response payloadLabel] isEqualToString:@"federation-not-available"]) {
+        response.transportSessionError = nil;
+    }
```

Comment:
what about moving this to transportErrorFromURLTask:expired: with an additional param to create our own error?

Reply:
done